config/locales/en.yml

```diff
@@ -582,6 +583,8 @@ en:
       properties: "Marks Spreadsheet Properties"
       required_fields: "Required Fields <span class='required_field'>*</span>"
       show_total: "Show Total"
+      id_number: "Student Number"
```

Comment:
Good idea, but please don't add new keys. Instead, use the existing labels for the attributes under `user` (roughly line 963).